packages/semaphore-signature-pcd/src/SemaphoreSignaturePCD.ts

```diff
@@ -119,7 +132,7 @@ export async function prove(
 
   // Set up singleton group
   const group = new Group(1, 16);
-  group.addMember(identityPCD.claim.identity.commitment);
+  group.addMember(identityPCD.claim.identity.commitment + "");
```

Comment:
Why is this necessary?

Reply:
This one isn't necessary, but other ones are, in order to convert from `BigNumberish` (which works in semaphore) to `string` (which is the types we have chosen to represent big numbers in the `claim` of these PCDs